newtests/lsp/completion/test.js

```diff
@@ -176,5 +176,70 @@ export default suite(
         [...lspIgnoreStatusAndCancellation],
       ),
     ]),
-  ],
+    test('textDocument/completion', [
+      addFile('params.js'),
+      ideStartAndConnect(),
+      ideRequestAndWaitUntilResponse('textDocument/completion', {
+        textDocument: {uri: '<PLACEHOLDER_PROJECT_URL_SLASH>params.js'},
+        position: {line: 9, character: 15},
+        context: { triggerKind: 1 }
+      }).verifyAllIDEMessagesInStep(
```

Comment:
I suspect this test is asserting that, for each completion item, these fields are correctly set. However, it doesn't assert that these are ALL the fields. So even though this test is passing, some of these completion items have other fields too.

I suspect this because the relevant fields are missing here. Like I think there should at least be a `textEdits` field which specifies what the IDE should autofill in. And since that's the thesis of this diff, the test should probably check it.

Can you update the test? Or let me know if I'm wrong? Thanks!

Reply:
it turns out I really wasn't sending the textEdit response. but that was because in the test, I didn't set `snippetSupport: true`. I fixed that and have now updated the tests.